browser/scripts/application.js

```diff
@@ -2178,6 +2198,16 @@ Application.prototype.setupChat = function() {
 
 	this.chatStore = new E2.ChatStore()
 	this.chat = new E2.Chat($('#chat'))
+	$('.chat-users').draggable({
```

Comment:
instead of doing it this way, would you mind taking our scripts/draggable and using that? it could use a slight refactor (now it's making a global `make_draggable()` which could just be namespaced under E2.app)